extensions/sql-migration/src/api/utils.ts

```diff
@@ -218,3 +218,13 @@ export function getMigrationStatusImage(status: string): IconPath {
 			return IconPathHelper.error;
 	}
 }
+
+export function get12HourTime(date: Date): string {
+	let hours = date.getHours();
+	const ampm = hours >= 12 ? 'pm' : 'am';
+	hours = hours % 12;
+	hours = hours ? hours : 12; // the hour '0' should be '12'
+	let minutes = date.getMinutes();
+	const minutesString = minutes < 10 ? '0' + minutes : minutes;
+	return hours + ':' + minutesString + ' ' + ampm;
+}
```

Comment:
would date.toLocaleString(locales?: string | string[], options?: Intl.DateTimeFormatOptions): string;
be more suitable for this?